### ISSUE TITLE
Write tmp files to OUTDIR + flag to keep tmp files

### ISSUE THREAD

**Issue by joegeorgeson** [SchwartzLab/brainSTORM#2]
It would be more convenient if tmp files were written to the OUTDIR and also a flag added indicating whether or not to keep tmp files.

**Comment by AngelCampos** [comment]
Yep definitely agree let's figure out which would be the optimal
